core/rfb.js

```diff
@@ -2534,9 +2574,24 @@ RFB.encodingHandlers = {
         this._FBU.bytes = pixelslength + masklength;
         if (this._sock.rQwait("cursor encoding", this._FBU.bytes)) { return false; }
 
-        this._cursor.change(this._sock.rQshiftBytes(pixelslength),
-                            this._sock.rQshiftBytes(masklength),
-                            x, y, w, h);
+        // Decode from pixels + mask to RGBA
+        let rgba = [];
```

Comment:
might want to use a typedarraybuffer here instead.

Reply:
This was already in the old code that I just moved from one place to the other.